src/h5web/toolbar/controls/DomainSlider.tsx

```diff
@@ -60,6 +51,18 @@ function DomainSlider(props: Props): ReactElement {
         step={step}
         pearling
       />
+
+      <ToggleBtn
+        className={styles.autoBtnLike}
+        label="auto-scale"
+        icon={FiZap}
```

Comment:
A button with a label of `auto` looked a bit ugly. I hope the `zap` icon is clear enough.

Reply:
To me, it is not :joy: !

What about keeping the previous icon with the auto label:
![image](https://user-images.githubusercontent.com/42204205/106145524-f0907900-6175-11eb-8114-033d13c1a92f.png)

I think `Auto` needs to be here for someone to understand the button's role

Reply:
Aww okay 😢 😂

Reply:
Not sure the previous icon makes much sense with just "auto" as the label, though. What do you think?

Reply:
Nah, we can remove the icon but I found the button a bit bare without an icon. Want to put `zap` instead :zap: ?